src/app/lib/config/services/japanese.js

```diff
@@ -132,37 +131,41 @@ const service = {
     },
     brandSVG,
     mostRead: {
-      header: 'Most read',
+      header: '一番人気',
       lastUpdated: 'Last updated: ',
     },
     footer: {
       externalLink: {
         href: 'https://www.bbc.co.uk/help/web/links/',
-        text: 'Read about our approach to external linking.',
+        text: '本サイトの外部リンク方針はこちら',
       },
       links: [
         {
-          href: 'https://www.bbc.com/news/help-41670342',
-          text: 'Why you can trust the BBC',
+          href: 'https://www.bbc.com/japanese/help-49677253',
+          text: 'BBCが信頼できる理由',
+        },
+        {
+          href: 'https://www.bbc.com/japanese/help/terms',
+          text: '利用条件',
         },
         {
-          href: 'https://www.bbc.com/terms',
-          text: 'Terms of Use',
+          href: 'https://www.bbcworldnews-japan.com/about_us/',
+          text: 'BBCについて',
         },
         {
-          href: 'https://www.bbc.com/privacy/',
-          text: 'Privacy Policy',
+          href: 'https://www.bbc.com/japanese/help/privacy',
+          text: '個人情報取り扱い方針',
         },
         {
-          href: 'https://www.bbc.com/usingthebbc/cookies/',
-          text: 'Cookies',
+          href: 'https://www.bbc.co.uk/usingthebbc/cookies/',
+          text: 'クッキー',
         },
         {
-          href: 'https://www.bbc.com/contact/',
-          text: 'Contact the BBC',
+          href: 'https://www.bbc.co.uk/contact',
+          text: 'BBCに連絡する',
```

Comment:
This is now `Contact the BBC` on the Footer tab in the spreadsheet, but it had a translation (the one here) before